pkg/logql/evaluator.go

```diff
@@ -207,6 +207,12 @@ func (ev *DefaultEvaluator) StepEvaluator(
 		return binOpStepEvaluator(ctx, nextEv, e, q)
 	case *syntax.LabelReplaceExpr:
 		return labelReplaceEvaluator(ctx, nextEv, e, q)
+	case *syntax.VectorExpr:
+		val, err := e.Value()
+		if err != nil {
+			return nil, err
```

Comment:
Can you please add a test-case where an unexpected value is used, to cover this case?

Reply:
ok,done.

Reply:
ok,done

Reply:
Thanks a lot!